inventory/group_vars/be.yml

```diff
@@ -0,0 +1,26 @@
+---
+
+# See https://github.com/openfoodfoundation/ofn-install/wiki/Setup for more info
+checkout_zone: Europe
+country_code: BE
+currency: EUR
+locale: fr
+language: en_US.UTF-8
+language_packages:
+  - language-pack-en-base
+  - language-pack-fr-base
+l10n_repo: https://github.com/openfoodfoundation/l10n_fr.git
+timezone: Brussels
+
+# Used for the LetsEncrypt email setting
+developer_email: admin@openfoodnetwork.be
+
+# Users listed here will have their key from /files/keys added to the unicorn_user
+users_sysadmin:
+  - paco
+  - maikel
+  - matt
+  - pau
+  - enricostn
+  - luis
+  - kristina
```

Comment:
I think soon we'll have to extract a `core_dev_team` in a var and use it in all `users_sysadmin`.